### PR TITLE
Pr/harmonize editable value

### PR DESCRIPTION
**Pull request by niklaushug**

### CODE REVIEW

File: packages/tocco-ui/src/EditableValue/typeEditors/PhoneEdit.spec.js [hunk 1]
```diff
@@ -1,24 +1,18 @@
...
-        test('should display the formatted phone number in input', () => {
-          const wrapper = shallow(<PhoneEdit value="+41794733123" onChange={() => {}}/>)
-          return wrapper.instance().importLibPhoneNumber()
-            .then(() => expect(wrapper.find('input').props()['value']).to.eql('+41 79 473 31 23'))
-        })
-
         test('should call on change with phone number in e.164 format', () => {
           const onChangeSpy = sinon.spy()
-          const wrapper = shallow(<PhoneEdit value="+41794733123" onChange={onChangeSpy}/>)
+          const wrapper = mount(<PhoneEdit value="0792345678" onChange={onChangeSpy}/>)
```
Comment (MethodenMann) [inline packages/tocco-ui/src/EditableValue/typeEditors/PhoneEdit.spec.js]: just out of curiosity: why did you change the tests data?

Comment (niklaushug) [reply packages/tocco-ui/src/EditableValue/typeEditors/PhoneEdit.spec.js]: Changing the telephone number helped me debugging. IMO a logical sequence of figures increases readability and looks less like a real telephone number.
